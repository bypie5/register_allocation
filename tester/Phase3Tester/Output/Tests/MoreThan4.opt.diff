--- SelfTestCases/MoreThan4.opt.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/MoreThan4.opt.out	2020-03-09 14:31:09.000000000 -0700
@@ -1,13 +1,3 @@
-1
-2
-3
-4
-5
-6
-6
-5
-4
-3
-2
-1
-0
+--- Error (ProgramFault) ---
+require function address, found undefined value
+  Main() line 13
