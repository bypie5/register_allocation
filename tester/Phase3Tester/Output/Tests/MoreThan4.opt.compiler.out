func Main [in 0, out 0, local 0]
$s0 = 
PrintIntS($s0)
ret
func MT4.Start [in 0, out 0, local 0]
PrintIntS($s0)
PrintIntS($s4)
PrintIntS($s1)
PrintIntS($s5)
PrintIntS($s2)
PrintIntS($s6)
$s7 = 
ret
func MT4.Change [in 0, out 0, local 0]
PrintIntS($s0)
PrintIntS($s1)
PrintIntS($s2)
PrintIntS($s3)
PrintIntS($s4)
PrintIntS($s5)
ret

